saleor/graphql/payment/types.py

```diff
@@ -43,14 +43,30 @@
 
 
 class Transaction(ModelObjectType[models.Transaction]):
-    id = graphene.GlobalID(required=True)
-    created = graphene.DateTime(required=True)
-    payment = graphene.Field(lambda: Payment, required=True)
-    token = graphene.String(required=True)
-    kind = TransactionKindEnum(required=True)
-    is_success = graphene.Boolean(required=True)
-    error = graphene.String()
-    gateway_response = JSONString(required=True)
+    id = graphene.GlobalID(
+        required=True, description="ID associated with a transaction."
+    )
+    created = graphene.DateTime(
+        required=True, description="Date and time which transaction was created."
+    )
+    payment = graphene.Field(
+        lambda: Payment,
+        required=True,
+        description="Determines the payment associated with a transaction.",
+    )
+    token = graphene.String(
+        required=True, description="Unique token associated with a transaction."
+    )
+    kind = TransactionKindEnum(
+        required=True, description="Determines the type of transaction."
+    )
+    is_success = graphene.Boolean(
+        required=True, description="Determines if the payment was successful."
```

Comment:
Just to be more precise here. `is_success` deteremines if a specific payment action (`Transaction) was successful. 
It could an action like refund or charge etc. So I would say that we should have: `the payment` -> `the transaction`